.devcontainer/Dockerfile

```
@@ -1,4 +1,15 @@
-FROM mcr.microsoft.com/devcontainers/go:1.22-bookworm
+ARG GO_VERSION=1.24.1
+
+# ビルド変数で指定したバージョンのgolangを取得
+FROM alpine:latest AS latest-go-fetcher
```

Hadolint (2.12.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)